ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -711,6 +711,10 @@ ASYNC_INDICATORS_TO_QUEUE = {{ localsettings.ASYNC_INDICATORS_TO_QUEUE }}
 ASYNC_INDICATOR_QUEUE_CRONTAB = crontab(**{{ localsettings.get('ASYNC_INDICATOR_QUEUE_CRONTAB') }})
 {% endif %}
 
+{% if localsettings.get('ASYNC_INDICATOR_QUEUE_TIMES') %}
+ASYNC_INDICATOR_QUEUE_TIMES = crontab(**{{ localsettings.get('ASYNC_INDICATOR_QUEUE_TIMES') }})
```

Comment:
Since `ASYNC_INDICATOR_QUEUE_TIMES = {'*': [[11, 23], [0, 4]], '7': [[0, 24]]}`, does that mean that `crontab(**ASYNC_INDICATOR_QUEUE_TIMES)` is a very bizarre thing to do? I don't even think there's a way to write it literally/without the splat: in made-up syntax it would be `crontab('*'=[[11, 23], [0, 4]], '7'=[[0, 24]]}`.

Reply:
You're totally correct and I already fixed it here: https://github.com/dimagi/commcarehq-ansible/pull/1262

Reply:
Hahaha okay, sorry I missed that.